pandas/plotting/_core.py

```diff
@@ -1533,6 +1533,53 @@ def hexbin(self, x, y, C=None, reduce_C_function=None, gridsize=None, **kwargs):
         return self(kind="hexbin", x=x, y=y, C=C, **kwargs)
 
 
+_backends = {}
+
+
+def _find_backend(backend: str):
+    """
+    Find a pandas plotting backend>
+
+    Parameters
+    ----------
+    backend : str
+        The identifier for the backend. Either an entrypoint item registered
+        with pkg_resources, or a module name.
+
+    Notes
+    -----
+    Modifies _backcends with imported backends as a side effect.
+
+    Returns
+    -------
+    backend : types.ModuleType
```

Comment:
```suggestion
    types.ModuleType
```

We've been just naming the output when a function returns a tuple. In this case seems reasonable, but we had many "transform -> transformed" names that didn't add much value.